### ISSUE TITLE
minor spelling errors in binary

### ISSUE THREAD

**Issue by emollier** [veg/hyphy#1226]
Good day,

While working on packaging _hyphy_, _lintian_ caught a few spelling issues in the resulting executable:
```
I spelling-error-in-binary usr/bin/hyphy agrument argument
I spelling-error-in-binary usr/bin/hyphy arguement argument
I spelling-error-in-binary usr/bin/hyphy conjuction conjunction
I spelling-error-in-binary usr/bin/hyphy defaut default
I spelling-error-in-binary usr/bin/hyphy dependance dependence
I spelling-error-in-binary usr/bin/hyphy enviroment environment
I spelling-error-in-binary usr/bin/hyphy partion partition
I spelling-error-in-binary usr/bin/hyphy whcih which
```
I haven't checked toroughly were they might occur precisely yet, but some of those may pop up during program execution, so you might be interested in being aware of them.

Kind Regards,
Étienne.

**Comment by spond** [comment]
Dear @emollier,

Thanks for reporting! Will make a pass and fix the ones I find for the next patch release.

Best,
Sergei

**Comment by emollier** [comment]
Sergei Pond, on 2020-09-29 11:53:48 -0700:
> Thanks for reporting! Will make a pass and fix the ones I find for the next patch release.

Great!  You're welcome.  :)

**Comment by spond** [comment]
Dear @emollier,

These issues have been resolved in 2.5.19 release.

Best,
Sergei
